basic-example-imagespec/README.md

```diff
@@ -0,0 +1,25 @@
+# {{ cookiecutter.project_name }}
+
+A template for the recommended layout of a Flyte enabled repository for code written in python using [flytekit](https://docs.flyte.org/projects/flytekit/en/latest/).
+
+## Usage
+
+To get up and running with your Flyte project, we recommend following the
+[Flyte getting started guide](https://docs.flyte.org/en/latest/getting_started.html).
+
+This project includes a script `docker_build.sh` that you can use to build a
```

Comment:
Not sure if I'm too confused but, I don't understand if these instructions are related to this particular template

Reply:
Good flag, I forgot to remove this from the README.